posthog/hogql_queries/web_analytics/session_attribution_explorer_query_runner.py

```diff
@@ -11,7 +11,25 @@
     HogQLFilters,
 )
 
-BREAKDOWN_NULL_DISPLAY = "(none)"
+
+# NOTE: These must exist on `sessions_v1` and `sessions_v2` tables
```

Comment:
Ah this also means that they would need to be on the underlying clickhouse table, which they aren't for v1 here https://github.com/PostHog/posthog/blob/e81f5da0486806abb8653f13d5f88dede38eebc9/posthog/models/sessions/sql.py

I don't think it's worth updating the v1 table, it's probably easier to have 2 lists and check the HogQLModifiers sessionTableVersion

Reply:
I checked the `raw_sessions.py` file and assumed that if they were there then they'd be available for v1 as well? Yikes, will fix this, good catch